### ISSUE TITLE
Avoid extracting text from the figure in the article.

### ISSUE THREAD

**Issue by heweapon** [VikParuchuri/marker#228]
Hi. First of all, let me say that this is an awesome work! It helps me a lot!
But I have a problem, when the figures in the paper contain words, marker will extract these words and put them into Markdown, thus causing text confusion. May I ask what I should do to avoid recognizing the words in the figures?

**Comment by conjuncts** [comment]
I've noticed that newer versions of marker do indeed exclude words in figures. Do you have a sample PDF where this issue persists?

**Comment by MikelAhedo** [comment]
You're right, I'm suffering the same issue. When the figures contain text, sometimes marker tries to extract their content instead of adding the path ![ ](path).

It looks like if the model doesn't recognize that there are images and tries to process them as text.
